src/styles/blocks/star.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  margin: 0 2px;
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
One of this rules can be removed after you set margins to one side instead of both sides